pkg/proxy/iptables/proxier.go

```diff
@@ -169,6 +169,8 @@ type Proxier struct {
 	haveReceivedServiceUpdate   bool // true once we've seen an OnServiceUpdate event
 	haveReceivedEndpointsUpdate bool // true once we've seen an OnEndpointsUpdate event
 	throttle                    flowcontrol.RateLimiter
+	timer                       *time.Timer
+	lastSync                    time.Time
```

Comment:
We could leave this out of 1.5 entirely, or we could take a minimal patch for a 1.5 cherrypick followed by this sort of cleanup AND TESTING.  @timothysc what do you think?